phoenicis-javafx/src/main/java/org/phoenicis/javafx/components/common/skin/PhoenicisViewSkin.java

```diff
@@ -0,0 +1,81 @@
+package org.phoenicis.javafx.components.common.skin;
+
+import javafx.beans.binding.Bindings;
+import javafx.beans.binding.ObjectExpression;
+import javafx.scene.Node;
+import javafx.scene.control.ProgressIndicator;
+import javafx.scene.layout.BorderPane;
+import javafx.scene.layout.HBox;
+import org.phoenicis.javafx.components.common.control.DetailsPanel;
+import org.phoenicis.javafx.components.common.control.PhoenicisView;
+import org.phoenicis.javafx.components.common.control.SidebarBase;
+
+/**
+ * A base skin implementation for {@link PhoenicisView} components implementations
+ *
+ * @param <C> The control type of the component
+ * @param <S> The skin type of the component
+ */
+public abstract class PhoenicisViewSkin<C extends PhoenicisView<C, S>, S extends PhoenicisViewSkin<C, S>>
```

Comment:
I don't really like the class name. We should find a better description for what you call `PhoenicisView`. Maybe something like "page" or "tab", not sure currently.

Reply:
I don't want to use "tab", because this suggests an implementation of the `Tab` class, which I'm not doing. For "page" I'm missing a direct relation to the usage of the components.

Reply:
I understand, still I don't like the current naming. I don't really have a different idea though...

Reply:
I agree with you that the name is not ideal, but I think that we can solve this issue later and change the name of the class when we find a better fitting name.
What do you think?

Reply:
We don't really have another choice unless we find a better name now, have we? Maybe @qparis has an idea.

Reply:
We could also go with `PhoenicisPanel` or `PhoenicisFeature` or `PhoenicisFeaturePanel`.

Reply:
`FeaturePanel` sounds good to me. Why add `Phoenicis`?

Reply:
Should we also rename our current "views" to `FeaturePanel`, i.e. `ApplicationsView` -> `ApplicationFeaturePanel`? Later on I also want to rename the `MainWindow` class, should we rename it to `PhoenicisApplicationPanel`?

Reply:
Sounds reasonable.

Reply:
Done